crates/wasmtime/src/instance.rs

```diff
@@ -102,7 +104,47 @@ pub struct Instance {
     module: Module,
 }
 
+#[allow(dead_code)]
+pub struct HostState {
+    frame_info_registration: Option<Arc<GlobalFrameInfoRegistration>>,
```

Comment:
```suggestion
    _frame_info_registration: Option<Arc<GlobalFrameInfoRegistration>>,
```

This supressed the dead code warning for this field only.

Reply:
I moved `#[allow(dead_code)]` to above the `frame_info_registration` field